.eslintrc.json

```diff
@@ -112,7 +112,18 @@
             "error",
             "prefer-double"
         ],
-        "import/order": "error",
+        "import/order": [
+            "error", {
+                "groups": [
+                    "builtin",
+                    "external",
+                    "parent",
+                    "sibling",
+                    "index",
+                    "type"
```

Comment:
Type was added: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#groups-array so they are again at the end. In the last releases they did add it and then revert it. I did now set a specific rule so it will not longer change here and I think having `type` at the end will make it easier in future for us.